Automattic/pocket-casts-ios
# Consolidate Serchbars UI

| 📘 Part of: # |  
|:---:|

Fixes #2259  


Update font and colors on the Bookmarks Search Bar to match the ones in PCSearchBarController

| 1 | 2 | 3 | 4 | 5 |
| - | - | - | - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 58 09](https://github.com/user-attachments/assets/d213a994-0fd7-4c60-961d-7c833c906231) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 57 52](https://github.com/user-attachments/assets/8406044c-d7dd-46c2-8585-3e9280def375) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 57 32](https://github.com/user-attachments/assets/1087b77b-606e-4e5d-b583-69e4acbde3fd) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 57 17](https://github.com/user-attachments/assets/6a9d06e6-f88a-4f69-8720-47259c55ab61) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 57 04](https://github.com/user-attachments/assets/d0194a75-fc0f-4a3c-9fdf-c9dbc632561e) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 15 58](https://github.com/user-attachments/assets/b2ad37e6-132d-46ea-8872-4fd257c3ea22) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 16 57](https://github.com/user-attachments/assets/a5dd55d2-01c2-4635-8354-d2bdc1621f5c) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 16 31](https://github.com/user-attachments/assets/8276fce4-6c4f-477d-b4f9-2fb7612993f1) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 16 13](https://github.com/user-attachments/assets/d968ad61-794b-442f-8782-3c5f84f8bd85) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-12-13 at 13 17 16](https://github.com/user-attachments/assets/8f817669-c8a1-4fe1-baa6-f0e2664e164e) |


## To test

1. Start the app
2. Go to Profile -> Bookmarks
3. Check the Search Bar UI ( icons, text color, placeholder text, cancel button) and see if they match the search bar in Discovery and Podcasts
4. Open the Podcast page for a podcast that you have bookmarks
5. Tap on the bookmarks tab
6. Check if the search UI match the one in Discovery and Podcasts

## Checklist
